packages/story-editor/src/components/panels/document/taxonomies/taxonomies.js

```diff
@@ -48,10 +49,10 @@ function TaxonomiesPanel(props) {
   }
 
   const availableTaxonomies = taxonomies.filter((taxonomy) => {
-    const isVisible = taxonomy?.visibility?.show_ui;
+    const isVisible = taxonomy?.visibility?.showUi;
     const canAssignTerms = Boolean(
-      capabilities[`assign-${taxonomy?.restBase}`] ||
-        capabilities[`assign-${taxonomy?.slug}`]
+      capabilities[snakeToCamelCase(`assign-${taxonomy?.restBase}`)] ||
+        capabilities[snakeToCamelCase(`assign-${taxonomy?.slug}`)]
```

Comment:
Just confirming, this still works as expected?

Reply:
Yes, double-checked, and it works as expected.